mobile/components/QuestionCardFooter.tsx

```diff
@@ -26,14 +27,7 @@ const QuestionCardFooter: React.FC<QuestionCardFooterProps> = ({
           attachmentTypes &&
           attachmentTypes.length !== 0 &&
           attachmentTypes.map((attachmentType, index) => {
-            const icon =
-              attachmentType === AttachmentMimeType.IMG
-                ? "photo"
-                : attachmentType === AttachmentMimeType.VIDEO
-                  ? "video"
-                  : attachmentType === AttachmentMimeType.AUDIO
-                    ? "audio"
-                    : "photo";
+            const icon = useMemo(() => mapMimeTypeToIcon(attachmentType), [attachmentType]);
```

Comment:
Nu face nimic useMemo aici. Mai degraba QuestionCardFooter poate sa fie memo ca sa reactioneze doar la schimbarile de props. 

In cazul asta, attachmentType va fi mereu altul pentru ca itereaza peste acel array.

Reply:
done ✅